packages/react-next/src/components/Pivot/Pivot.base.tsx

```diff
@@ -190,15 +195,46 @@ export const PivotBase: React.FunctionComponent<IPivotProps> = React.forwardRef(
       return undefined;
     };
 
-    const { theme, linkSize, linkFormat } = props;
     classNames = getClassNames(props.styles!, {
       theme: theme!,
       linkSize,
       linkFormat,
     });
 
     const renderedSelectedKey = getSelectedKey();
-    const items = linkCollection.links.map(l => renderPivotLink(linkCollection, l, renderedSelectedKey));
+    const renderedSelectedIndex = renderedSelectedKey ? linkCollection.keyToIndexMapping[renderedSelectedKey] : 0;
+
+    const items = linkCollection.links.map(l =>
+      renderPivotLink(linkCollection, l, renderedSelectedKey, classNames.link),
+    );
+
+    const overflowMenuButtonComponentRef = React.useRef<IButton>();
+    const overflowMenuButtonRef = React.useRef<HTMLElement>();
+    const setOverflowMenuButtonRef = (button: ActionButton | null) => {
+      const node = ReactDOM.findDOMNode(button);
```

Comment:
This would not work once the buttons are migrated to function components in future, since ReactDOM.findDOMNode cannot be used on function components.

Reply:
The overflow algorithm won't work unless it can get to the DOM node to measure it. It looks like the new Button exposes a ref directly to the <button> Element, which is even better, and will simplify this code so it doesn't need to use findDOMNode. I'll leave this as-is for now until Pivot uses the new Button.